cluster/src/dunit/scala/org/apache/spark/sql/NorthWindDUnitTest.scala

```diff
@@ -508,128 +532,293 @@ class NorthWindDUnitTest(s: String) extends ClusterManagerTestBase(s) {
+    NWQueries.regions(snc).write.insertInto("regions")
 
     snc.sql(NWQueries.categories_table)
-    NWQueries.categories.write.insertInto("categories")
+    NWQueries.categories(snc).write.insertInto("categories")
 
     snc.sql(NWQueries.shippers_table)
-    NWQueries.shippers.write.insertInto("shippers")
+    NWQueries.shippers(snc).write.insertInto("shippers")
 
     snc.sql(NWQueries.employees_table +
         " using row options( partition_by 'EmployeeID', buckets '3')")
-    NWQueries.employees.write.insertInto("employees")
+    NWQueries.employees(snc).write.insertInto("employees")
 
     snc.sql(NWQueries.customers_table + " using column options(" +
         "partition_by 'CustomerID', buckets '19', redundancy '1')")
-    NWQueries.customers.write.insertInto("customers")
+    NWQueries.customers(snc).write.insertInto("customers")
 
     snc.sql(NWQueries.orders_table + " using row options (" +
         "partition_by 'CustomerID', buckets '19', " +
         "colocate_with 'customers', redundancy '1')")
-    NWQueries.orders.write.insertInto("orders")
+    NWQueries.orders(snc).write.insertInto("orders")
 
     snc.sql(NWQueries.order_details_table + " using row options (" +
         "partition_by 'ProductID', buckets '329', redundancy '1')")
-    NWQueries.order_details.write.insertInto("order_details")
+    NWQueries.order_details(snc).write.insertInto("order_details")
 
     snc.sql(NWQueries.products_table +
         " USING column options ( partition_by 'ProductID', buckets '329'," +
         " colocate_with 'order_details', redundancy '1')")
-    NWQueries.products.write.insertInto("products")
+    NWQueries.products(snc).write.insertInto("products")
 
     snc.sql(NWQueries.suppliers_table +
         " USING column options (PARTITION_BY 'SupplierID', buckets '123')")
-    NWQueries.suppliers.write.insertInto("suppliers")
+    NWQueries.suppliers(snc).write.insertInto("suppliers")
 
     snc.sql(NWQueries.territories_table +
         " using column options (partition_by 'TerritoryID', buckets '3')")
-    NWQueries.territories.write.insertInto("territories")
+    NWQueries.territories(snc).write.insertInto("territories")
 
     snc.sql(NWQueries.employee_territories_table + " using row options(" +
         "partition_by 'TerritoryID', buckets '3', colocate_with 'territories')")
-    NWQueries.employee_territories.write.insertInto("employee_territories")
+    NWQueries.employee_territories(snc).write.insertInto("employee_territories")
+  }
+
+  protected def getTempDir(dirName: String): String = {
+    val log: File = new File(".")
+    var dest: String = null
+    dest = log.getCanonicalPath + File.separator + dirName
+    val tempDir: File = new File(dest)
+    if (!tempDir.exists) tempDir.mkdir()
+    return tempDir.getAbsolutePath
+  }
+
+  def assertQueryFullResultSet(snc: SnappyContext, sqlString: String, numRows: Int, queryNum: String, tableType: String, pw: PrintWriter, sqlContext: SQLContext): Any = {
```

Comment:
Break line. Please take care of scalaStyle in our sources too. Currently scalaStyle checks for snappydata modules have to be disabled precisely because a very large number of violations in our code.